src/views/HomeView.tsx

```diff
@@ -1,9 +1,31 @@
 import HeaderBarWrapper from "@src/components/HeaderBarWrapper";
 import { NavBar } from "@src/components/NavBar";
 import { TopicCard } from "@src/components/PostCard";
-import { ScrollView, StyleSheet, Text, View } from "react-native";
+import { getAllTopics } from "@src/services/viewService/getTopics";
+import { useQuery } from "@tanstack/react-query/build/lib/useQuery";
+import { FlatList, StyleSheet, Text, View } from "react-native";
 
 export const HomeView: React.FC = () => {
+  const {
+    isLoading,
+    isError,
+    data: topicList,
+    error,
+  } = useQuery<ForumTopic[]>({
+    queryKey: ["allTopics", "..."],
```

Comment:
Well, I mean you should have something substituted to `"..."` 😂

Reply:
I really can't think of some unique description to add to this extensive collection. When we start to get the topic step by step with separate requests, we can add more information behind it.